spark/1.11/README.md

```diff
@@ -0,0 +1,160 @@
+
+**Table of Contents**:
+
+- [Prerequisites](#prerequisites)
+- [Install Spark](#install-spark)
+- [Launch a Spark job](#launch-a-spark-job)
+- [Uninstall Spark](#uninstall-spark)
+
+## Prerequisites
+
+- A running DC/OS 1.11 cluster with at least 3 nodes with each 2 CPUs and 2 GB of RAM available.
+- [DC/OS CLI](https://dcos.io/docs/1.11/cli/install/) installed.
+
+## Install Spark
+
+Assuming you have a DC/OS cluster up and running, the first step is to install Spark:
+
+```bash
+$ dcos package install spark
+By Deploying, you agree to the Terms and Conditions https://mesosphere.com/catalog-terms-conditions/#certified-services
+Continue installing? [yes/no] yes
+Installing Marathon app for package [spark] version [2.3.1-2.2.1-2]
+Installing CLI subcommand for package [spark] version [2.3.1-2.2.1-2]
+New command available: dcos spark
+DC/OS Spark is being installed!
+
+        Documentation: https://docs.mesosphere.com/service-docs/spark/
+        Issues: https://docs.mesosphere.com/support/
+```
+
+If the spark service is already installed and you only wish to install the CLI subcommand on your client, simply do:
+
+```bash
+$ dcos package install spark --cli
+By Deploying, you agree to the Terms and Conditions https://mesosphere.com/catalog-terms-conditions/#certified-services
+Continue installing? [yes/no] yes
+Installing CLI subcommand for package [spark] version [2.3.1-2.2.1-2]
+New command available: dcos spark
+```
+
+Note that while the DC/OS CLI subcommand `spark` is immediately available, it takes a few moments for Spark to start running in the cluster.
+
+To see the new possible with the new `spark` subcommand, you can use the following command:
+
+```bash
+$dcos spark --help
+```
+
+Now, we can validate if Spark is running and healthy, in the cluster itself. For this, go to the DC/OS UI and you should see Spark there under the `Services` tab:
+
+![Services](img/services.png)
+
+You can see the SparkUI page if you use the `dcos spark webui` command:
+
+![Spark Drivers page](img/spark-drivers.png)
+
+This page contains all the datas about jobs which are launched in the Spark cluster. We will see later how to get jobs' logs from their.
+
+## Launch a Spark job
+
+Now that you've set up Spark, it's time to launch your first Spark job. We will use one of the [existing examples](https://github.com/apache/spark/blob/master/examples/src/main/scala/org/apache/spark/examples/SparkPi.scala) that comes with Spark.
+
+
+```bash
+cos spark run --submit-args='--driver-cores 1 --driver-memory 1024M --class org.apache.spark.examples.SparkPi https://downloads.mesosphere.com/spark/assets/spark-examples_2.10-1.4.0-SNAPSHOT.jar 30'
```

Comment:
Want to add the $ here like previous command examples? Also missing the "d" in dcos to begin this command.